storybook/storybook-react/src/HeroImage/HeroImage.stories.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Grid } from '@amsterdam/design-system-react'
+import { SearchField } from '@amsterdam/design-system-react'
+import { HeroImage } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Media/Hero Image',
+  component: HeroImage,
+  args: {
+    children: <SearchField />,
+  },
+} satisfies Meta<typeof HeroImage>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    src: 'https://picsum.photos/1280/360?random=1',
```

Comment:
All of these are suitable for work right? Maybe we could add some demo images in the storybook theme config to align with Figma.

Reply:
I expect them to be, yes. Having a couple of our own images instead is a good idea.